FxAClient/Frontend/SignIn/FxAContentViewController.swift

```diff
@@ -139,6 +139,12 @@ class FxAContentViewController: SettingsContentViewController, WKScriptMessageHa
 
     // Dispatch webkit messages originating from our child webview.
     func userContentController(userContentController: WKUserContentController, didReceiveScriptMessage message: WKScriptMessage) {
+        // Make sure we're communicating with a trusted page. That is, ensure the origin of the
```

Comment:
Please add a note about `securityOrigin`, and why you're testing as you are.  Worth including the details of our conversation on IRC about caveats, etc.